app/containers/SearchTrackContainer/index.js

```diff
@@ -73,15 +118,28 @@ export function SearchTrackContainer({
 
   const debouncedHandleTrackSearch = useCallback(debounce(handleTrackSearch, 500), []);
 
+  const tracks = get(tracksData, 'results');
+
+  const handleGlobalClick = (ref) => {
+    setCurrentTrack(ref);
+    const trackPaused = currentTrack?.current?.paused;
+    if (!trackPaused && ref?.current.src !== currentTrack?.current.src) {
+      currentTrack?.current?.pause();
+    }
+  };
+
+  useEffect(() => {
+    handleGlobalClick;
+  }, []);
```

Comment:
why is this used ?

Reply:
I will remove it as it is not required as per the current structuring of the container